### DISCUSSION TITLE
More than one discount for an event

### DISCUSSION THREAD

**Discussion by gdornez** [liedekef/events-made-easy#303] [Q&A] [answered]
I am faced with the following problem: our members get a discount per registered person. Our sponsors, however, get this discount for a maximum of two people. How do I handle this?
Kind regards,

Geert Dornez

**Comment by liedekef** [comment, marked as answer]
You can assign a discount group to an event, and add all configured/wanted discounts to that discount group.

**Comment by gdornez** [comment]
I created two discounts and placed them in a group called "Souper".
I then attached this group to the event. When using one of the discount codes, I get "You have not entered a valid discount code".
What am I doing wrong.
Kind regards

**Reply by gdornez** [reply]
Forget my last post. Everything works.
Kind regards,

Geert Dornez
